File: w08/d01/random_user/server_two.js
var request = require('request');
var express = require('express')
var app = express();

var url = "http://api.randomuser.me/";

app.get('/', function(req, res){

request(url, function (error, response, body) {
  if (!error && response.statusCode == 200) {
    data = JSON.parse(body)

    //user = data.results[0].user

    var name = data['results'][0]['user']['name']['first'];
    var email = data['results'][0]['user']['email'];
    var username = data['results'][0]['user']['username']

    var hash = {
    	name : name,
    	email : email,
    	username : username
    }
  }
  	// res.send(JSON.stringify(hash));
  	res.send(hash);
})

});

app.listen(3000);

console.log('listening on port 3000!');

File: w08/d01/random_user/server_two.rb
require 'httparty'
require 'sinatra'
require 'json'

get '/' do 
	url = "http://api.randomuser.me"
	response = HTTParty.get(url)
	content_type :json
	user = response['results'][0]['user']
	data =  {first: user['name']['first']
			last: user['name']['last']
			email: user['email']
			}
	data.to_json
end